app/translations/en-US.json

```diff
@@ -104,5 +104,13 @@
 	"Edit Shortcuts": "Edit Shortcuts",
 	"View Shortcuts": "View Shortcuts",
 	"History Shortcuts": "History Shortcuts",
-	"Window Shortcuts": "Window Shortcuts"
+	"Window Shortcuts": "Window Shortcuts",
+	"Add a Zulip organization": "Add a Zulip organization",
```

Comment:
I think you committed this file by mistake. Would you mind unstaging this?

Reply:
Yaa I will be unstaging it.